main/Cargo.toml

```diff
@@ -26,5 +26,4 @@ env_logger = "0.8.3"
 android-logd-logger = "0.1.3"
 
 [features]
-default = []
-hello-world = ["northstar/hello-world"]
\ No newline at end of file
+default = []
```

Comment:
Think a empty default set can just be removed along with the whole `[features]` section.

Reply:
I deleted `default` but how can we get rid of the whole `features` section? Those features are references in `lib.rs`. Should we get rid of these features everywhere?